src/WebJobs.Script.WebHost/Handlers/SystemTraceHandler.cs

```diff
@@ -67,14 +72,16 @@ private TraceWriter TraceWriter
                 { "uri", request.RequestUri.LocalPath.ToString() },
                 { "authorizationLevel", request.GetAuthorizationLevel().ToString() }
             };
-            TraceWriter.Info($"Executed HTTP request: {details}");
+            traceProperties[ScriptConstants.TracePropertyActivityIdKey] = details["requestId"];
```

Comment:
The code I previously wrote here is a little clunky, leading you to copy the pattern of resetting values that don't change. Can you change this so both the details and trace objects are created only once above before we log the first event, then for the "executed" event here you don't need to update traceProperties, and just add authorizationLevel to the details before logging again (the only value that changed).